.vscode/cspell.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": "0.1",
+  "language": "en_US",
+  "languageId": "csharp",
+  "dictionaries": ["powershell"],
+  "ignorePaths": [
+    "**/SessionRecords/**",
+    "**/packages/**",
+    "artifacts/**/*",
+    "*.exe",
+    "*.dll",
+    "*.nupkg",
+    "*.bmp",
+    "*.png",
+    "*.gif",
+    "*.jpg",
+    "*.tiff",
+    "*.wav",
+    "*.pdf",
+    "*.zip",
+    "*.sha512",
+    "*.ico",
+    "*.pri"
+  ],
+  "words": [],
```

Comment:
Oh, you should also put a comment in here that says to keep words sorted lexigraphically. Off hand, I don't remember if cSpell uses a case-sensitive sort or not, but it's probably on their site.

Reply:
The cspell docs don't call out a requirement for sorting but it's something I've been doing in other word lists for ease of use. Docs do say that cspell is case insensitive and I'll add that to the comment as well.

Reply:
Seems cSpell is not. I've seen others that are, so doesn't seem an issue.